crates/turborepo-lib/src/shim.rs

```diff
@@ -677,70 +678,21 @@ fn is_turbo_binary_path_set() -> bool {
     env::var("TURBO_BINARY_PATH").is_ok()
 }
 
-fn init_env_logger(verbosity: usize) {
-    // configure logger
-    let level = match verbosity {
-        0 => LevelFilter::Warn,
-        1 => LevelFilter::Info,
-        2 => LevelFilter::Debug,
-        _ => LevelFilter::Trace,
+fn init_subscriber(verbosity: usize) {
+    let max_level = match verbosity {
+        0 => LevelFilter::WARN,
+        1 => LevelFilter::INFO,
+        2 => LevelFilter::DEBUG,
+        _ => LevelFilter::TRACE,
     };
 
-    let mut builder = Builder::new();
-    let env = Env::new().filter("TURBO_LOG_VERBOSITY");
-
-    builder
-        // set defaults
-        .filter_level(level)
-        .write_style(WriteStyle::Auto)
-        // override from env (if available)
-        .parse_env(env);
-
-    builder.format(|buf, record| match record.level() {
-        Level::Error => {
-            let mut level_style = buf.style();
-            let mut log_style = buf.style();
-            level_style.set_bg(Color::Red).set_color(Color::Black);
-            log_style.set_color(Color::Red);
-
-            writeln!(
-                buf,
-                "{} {}",
-                level_style.value(record.level()),
-                log_style.value(record.args())
-            )
-        }
-        Level::Warn => {
-            let mut level_style = buf.style();
-            let mut log_style = buf.style();
-            level_style.set_bg(Color::Yellow).set_color(Color::Black);
-            log_style.set_color(Color::Yellow);
-
-            writeln!(
-                buf,
-                "{} {}",
-                level_style.value(record.level()),
-                log_style.value(record.args())
-            )
-        }
-        Level::Info => writeln!(buf, "{}", record.args()),
-        // trace and debug use the same style
-        _ => {
-            let now = Local::now();
-            writeln!(
-                buf,
-                "{} [{}] {}: {}",
-                // build our own timestamp to match the hashicorp/go-hclog format used by the go
-                // binary
-                now.format("%Y-%m-%dT%H:%M:%S.%3f%z"),
-                record.level(),
-                record.target(),
-                record.args()
-            )
-        }
-    });
-
-    builder.init();
+    // respect TURBO_LOG_VERBOSITY env var
```

Comment:
is this new? i didn't know this was a thing

Reply:
All I mean is that tracing lets you output logs to a bunch of different places; stdout, files, [opentelemetry](https://docs.rs/tracing-opentelemetry/latest/tracing_opentelemetry/) and even [chrome tracing](https://docs.rs/tracing-chrome/latest/tracing_chrome/) through the same logging / span infrastructure :)

Reply:
Our default 'subscriber' is one that prints to stdout following the same format as our go code.